dataland-frontend/src/components/general/DetailsCompanyDataTable.vue

```diff
@@ -14,6 +14,9 @@
           <ul v-if="Array.isArray(data[keyOfColumn])">
             <li :key="el" v-for="el in data[keyOfColumn]">{{ el }}</li>
           </ul>
+          <div v-else-if="data[keyOfColumn].quality">
```

Comment:
Since we want to use the table for the view only for DataPoint, we recognize it by that it is an object and contains quality

Reply:
Ok, and i will close it. But if in the future other objects with a property called quality would be introduced, this would have to be changed to check for more properties of a datapoint